rotkehlchen/tests/unit/test_assets.py

```diff
@@ -317,10 +317,9 @@ def test_case_does_not_matter_for_asset_constructor():
     assert a1.identifier == 'BTC'
     assert a2.identifier == 'BTC'
 
-    # We use a token that is only in ethereum (redacted)
-    a3 = symbol_to_ethereum_token('BTRFLY')
-    a4 = symbol_to_ethereum_token('BtRfLy')
-    assert a3.identifier == a4.identifier == strethaddress_to_identifier('0xC0d4Ceb216B3BA9C3701B291766fDCbA977ceC3A')  # noqa: E501
+    a3 = symbol_to_ethereum_token('DAI', chain_id=ChainID.ETHEREUM)
```

Comment:
This looks wrong. `symbol_to_ethereum_token` with a `chain_id` argument? :raised_eyebrow:

Reply:
Removed the chain_id parameter for now since it was not used anywhere anyway